docs/Developers/Writing_Good_Commits.rst

```diff
@@ -26,6 +26,9 @@ Example of a good commit:
 - `Fixes https://github.com/coala/coala/issues/5861`: Mention the URL
    of the issue it closes or fixes.
 
+Use 'Fixes' when you fixed a bug and 'Closes' when your work is
```

Comment:
I think we could clarify what a feature vs a bug is. For example, are we only considering user facing issues a bug? Currently, I think our definition is "anything which causes unexpected behavior for the users"